### ISSUE TITLE
Default dropout of `Gpt2Config` is 0.1?

### ISSUE THREAD

**Issue by anh-tong** [stanford-crfm/levanter#413]
Hi,

I'm curious about why the default dropout for `Gpt2Config` is set to 0, while HuggingFace config defaults is 0.1.

Based on my experience with Levanter, I found that setting the dropout to 0.1 improves model performance.

**Comment by anh-tong** [comment]
I'm sorry for not being clear about performance. What I meant is the validation perplexity. You can see the different between the two cases like following 
![image](https://github.com/stanford-crfm/levanter/assets/20637355/34f9d327-f8eb-4073-8fdd-a69034fc430d)
Anyway, thanks for your input on how RNG in JAX might affect the training time.

**Comment by dlwh** [comment]
sorry, it was me who wasn't clear. I understood you just fine! I was agreeing that yes sometimes it leads to worse loss, but typically only when we're data limited (in my experience), and the perf hit is substantial, so I set the default for the big data case.

Is this on openwebtext or wikitext or??? I'm shocked to be seeing signs of overfitting that quickly, though that's so early in the optimization (I would think?) that I don't really think you can draw any conclusions

**Comment by anh-tong** [comment]
This is trained on wikitext. I get it now. The amount of training data is small so the model tends to overfit. Thanks!
